install/install.md

```diff
@@ -10,23 +10,91 @@ description: |
 # RAPIDS Installation Guide
 {: .fs-8 }
 
-RAPIDS has several methods for installation, depending on the preferred environment and versioning. Returning users can jump straight to our **[release selector](#selector)** and new users can get started with these four steps:
+RAPIDS has several methods for installation, depending on the preferred environment and versioning. **New Users** should be mindful of the system and environment prerequisites. 
 {: .fs-6 .fw-300 }
 
-**[1. Provision System](#provision)** <br/>
-**[2A. Set Up Environment](#set-up)** <br/>
-**[2B. Set Up WSL2 Environment](#WSL2)** <br/>
-**[3A. Install RAPIDS (Release Selector)](#selector)** <br/>
-**[3B. Install RAPIDS (pip)](#pip-install)** <br/>
-**[4. Getting Started](#start)**  
+**[Install RAPIDS with Release Selector](#selector)** <br/>
+- [Installation Troubleshooting](#troubleshooting)
+
+**[System Requirements](#system-req)** <br/>
+- [OS, NVIDIA GPU Driver, and CUDA Versions](#system-req)
+- [(Optional) System Recommendations](#system-recommendations)
+- [(Optional) Cloud Instance GPUs](#cloud-gpu)
+
+**[Environment Setup](#environment)** <br/>
+- [Conda](#conda)
+- [Docker](#docker)
+- [Pip](#pip)
+- [Within WSL2](#wsl2)
+	- [Conda](#wsl2-conda)
+	- [Docker](#wsl2-docker)
+	- [Pip](#wsl2-pip)
+- [Build From Source](#source)
+
+**[Next Steps](#next-steps)**  
 
 <hr/>
+<div id="selector"></div>
+
+# Install RAPIDS
+Use the selector tool below to select your preferred method, packages, and environment to install RAPIDS. Certain combinations may not be possible and are dimmed automatically. 
+
+{% include selector.html %}
+
+<div id="troubleshooting"></div>
+
+## Install Troubleshooting
+
+### **PiP Issues**
```

Comment:
Can we order these consistently across the whole page? I prefer "Conda, pip, Docker" but it seems "Conda, Docker, pip" is the dominant order right now.
```suggestion
### **pip Issues**
```